galaxyenv/templates/nginx/wiki_hub_rewrites.conf.j2

```diff
@@ -1,5 +1,5 @@
 ##
-## This file is maintained by Ansible - CHANGES WILL BE OVERWRITTEN
+## Maps URLs from the old Moin-based wiki to the new metalsmith Galaxy Hub.
```

Comment:
The old comment is still true though. Mainly it's a reminder for me or anyone else who stumbles across the file when deployed on the server not to make any changes by hand because they'll just be overwritten by Ansible. However, a descriptive heading about the file contents is great. Can I suggest just using both?:

```
##
## This file is maintained by Ansible - CHANGES WILL BE OVERWRITTEN
##
## Maps URLs from the old Moin-based wiki to the new metalsmith Galaxy Hub.
##
```

Reply:
@natefoo I'll make the change and update the branch / PR.

Can I change the word "maintained" to "deployed".  Every time I come across this statement I try to look upstream to what file Ansible builds this file from.

Reply:
Done.  And yep, I'm OK manually testing some of these after deployment.

Reply:
Deployed is good. Now just go and change it in the hundreds of other files across our various playbooks and roles... ;D

Nah, this is good. Thanks!